Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -745,13 +740,59 @@ public IEnumerator JoystickActions_CanDriveUI()
         rightChildReceiver.Reset();
     }
 
+    [Test]
+    [Category("Actions")]
+    public void Actions_CanReassignUIActions()
+    {
+        var go = new GameObject();
+        go.AddComponent<EventSystem>();
+        var uiModule = go.AddComponent<InputSystemUIInputModule>();
+
+        const string kActions = @"
+            {
+                ""maps"" : [
+                    {
+                        ""name"" : ""Gameplay"",
+                        ""actions"" : [
+                            { ""name"" : ""Point"" }
```

Comment:
Maybe add a Gameplay/Navigate here, so we also test to make sure that reassigning will pick the right one?

Reply:
Done